### DISCUSSION TITLE
Use a object storage instead of local disk/drive

### DISCUSSION THREAD

**Discussion by mahabubx7** [foyzulkarim/mern-video-streaming#15] [Ideas]
Hi 👋🏻, I'm Mahabub. A bachelor student (CS) and web application developer (junior).

I have an idea to use an object storage for this R&D project. It can be something like AWS S3.

_I know a open source tool_ that can be an alternative to AWS S3.
It's called `MinIO`. It's also available in docker registry (image).

`MinIO` is already famous in the open source community for self hosted services. 

_I just shared my idea about this project and wish to contribute also (if I can do with my limited expertise)_

**Comment by foyzulkarim** [comment]
Thanks brother for the suggestion.
I have checked the MinIO, and it appears that they don't provide MIT license.
I don't want to attach with anything which has licensing issues. Sorry.

You are most welcome to contribute. I will add issues / tasks within next 1 or 2 weeks. 
Thanks.
